lib/drivers/tandemDriver.js

```diff
@@ -821,6 +821,12 @@ module.exports = function (config) {
 
   var listenForPacket = function (command,args,callback) {
 
+    var abortTimer = setTimeout(function () {
+      clearInterval(listenTimer);
+      debug('TIMEOUT');
```

Comment:
We have the exact same debug statement in several of the other drivers. For debug statements that are more "chatty", detailed logging is only available if `DEBUG_ERROR` environment variable is true, which enables debug statements wrapped in `if(__DEBUG__)` statements.